Gemfile

```diff
@@ -91,6 +91,11 @@ group(:omnibus_package) do
   gem "winrm-fs"
   gem "winrm-elevated"
   gem "cucumber"
+
+  # TODO Pinning these for now because github_changelog_generator has a bunch
+  # of different versions across our products
+  gem "nokogiri", "~> 1.6.3"
```

Comment:
Yup! We're going to get this merged since it got a green build and deal with the problem dependencies in another PR